source/using-ethereum-the-basics/ether.rst

```diff
@@ -5,27 +5,58 @@ Ether
 What is Ether?
 ================================================================================
 
+Ether is the base unit of currency used within Ethereum.  It is used to pay for
```

Comment:
Good point. I think we should refer to it as the crypto fuel required to run the blockchain or something similar.